azure-iot-device/azure/iot/device/common/auth/sastoken.py

```diff
@@ -19,7 +19,7 @@ class SasTokenError(ChainableException):
     pass
 
 
-class SasToken(object):
+class RenewableSasToken(object):
```

Comment:
>RenewableSasToken [](start = 6, length = 17)

Is Sas Token one word or two.  Here, 2 is implied, but the update_sasstoken implies that it's one word.

Reply:
We're kind of stuck with 'sastoken' as one word on the API since the addition of the 'sastoken_ttl' kwarg. Internally it's generally referred to as two words. Annoying, but not the end of the world I dont think?

---
In reply to: [506675813](https://github.com/Azure/azure-iot-sdk-python/pull/692#discussion_r506675813) [](ancestors = 506675813)

Reply:
why not make it Sastoken for this class then ?

---
In reply to: [506711196](https://github.com/Azure/azure-iot-sdk-python/pull/692#discussion_r506711196) [](ancestors = 506711196,506675813)